test/extended/builds/start.go

```diff
@@ -51,6 +51,25 @@ var _ = g.Describe("[builds][Slow] starting a build using CLI", func() {
 		})
 	})
 
+	g.Describe("setting OwnerReferences", func() {
+		g.It("should set parent BuildConfig as OwnerReference", func() {
+			g.By("starting the build from buildconfig")
+			br, err := exutil.StartBuildAndWait(oc, "sample-build")
```

Comment:
no need to wait for the build to finish, it'll be created immediately so you can just grab it and check.